OpenRA.Mods.Common/Traits/Attack/AttackFollow.cs

```diff
@@ -93,7 +94,8 @@ public override Activity Tick(Actor self)
 				if (self.IsDisabled())
 					return this;
 
-				var weapon = attack.ChooseArmamentsForTarget(target, forceAttack).FirstOrDefault();
+				var attack = SelectActiveAttack(target, forceAttack).FirstOrDefault();
+				var weapon = attack.Value;
```

Comment:
`if (attack != null)` throws me a syntax error. Should I check for the key instead of the value? (Because there is a nullcheck in the next line.)